### PR TITLE
fix(docs): replace AsyncAPI spec urls with valid ones

### LINKED ISSUE #22066: 🐛 Bug Report: Async API docs leads to a 404
### 📜 Description

Async API docs leads to a 404

### 👍 Expected behavior

Should link to the [Async API Spec](https://www.asyncapi.com/docs/reference/specification/v3.0.0)

### 👎 Actual Behavior with Screenshots

The URL 404s https://www.asyncapi.com/docs/specifications/latest/
image

### 👟 Reproduction steps

Click on the Async API links.

### 📃 Provide the context for the Bug.

_No response_

### 🖥️ Your Environment

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by RiverPhillips**
## Hey, I just made a Pull Request!

Replace AsyncAPI spec urls with valid ones. Resolves #22066 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/api-docs/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The plugin provides a standalone list of APIs, as well as an integration into th
 Right now, the following API formats are supported:
 
 - [OpenAPI](https://swagger.io/specification/) 2 & 3
-- [AsyncAPI](https://www.asyncapi.com/docs/specifications/latest/)
+- [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/v3.0.0)
```
Comment (tudi2d) [inline plugins/api-docs/README.md:16]:
```suggestion
- [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/latest)
```

File: docs/features/software-catalog/descriptor-format.md [hunk 2]
```diff
@@ -782,7 +782,7 @@ Describes the following entity kind:
 
 An API describes an interface that can be exposed by a component. The API can be
 defined in different formats, like [OpenAPI](https://swagger.io/specification/),
-[AsyncAPI](https://www.asyncapi.com/docs/specifications/latest/),
+[AsyncAPI](https://www.asyncapi.com/docs/reference/specification/v3.0.0),
```
Comment (tudi2d) [inline docs/features/software-catalog/descriptor-format.md:785]:
```suggestion
[AsyncAPI](https://www.asyncapi.com/docs/reference/specification/latest),
```

File: docs/features/software-catalog/descriptor-format.md [hunk 3]
```diff
@@ -838,7 +838,7 @@ The current set of well-known and common values for this field is:
 - `openapi` - An API definition in YAML or JSON format based on the
   [OpenAPI](https://swagger.io/specification/) version 2 or version 3 spec.
 - `asyncapi` - An API definition based on the
-  [AsyncAPI](https://www.asyncapi.com/docs/specifications/latest/) spec.
+  [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/v3.0.0) spec.
```
Comment (tudi2d) [inline docs/features/software-catalog/descriptor-format.md:841]:
```suggestion
  [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/latest) spec.
```

**Review by tudi2d — CHANGES_REQUESTED** [review]
thanks for fixing this! let's use `latest` over a specific version:)

**Review by tudi2d — APPROVED** [review]
ah, sorry - small addition. I believe you can remove the changeset as this PR only updates documentation.

other than that it looks good to me.

**Comment by RiverPhillips** [comment]
All done, thanks for the quick turnaround

**Comment by RiverPhillips** [comment]
Is it possible to re-run CI? Seems to be an issue in the node runtime, I don't think it's related to my changes

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks :pray:

**Pull request merged**